locale/en/get-involved/index.md

```diff
@@ -42,3 +42,4 @@ The Node.js community is large, inclusive, and excited to enable as many users t
 - [Spanish language Facebook group for Node.js](https://www.facebook.com/groups/node.es/)
 - [Spanish language community](http://nodehispano.com)
 - [Việt Nam Node.js community](http://nodejs.vn)
+- [Node.js Discord community](https://discord.gg/Uaq5RHu) - (unofficial)
```

Comment:
* This seems to be in the wrong place.  This would be a better fit in the "Community Discussion" section, it seems, although only because of the oddball inclusion of NodeUp in that section. TBH, this whole page seems a bit of a hodgepodge. It's not clear what kind of getting involved is included on this page and what kind of getting involved isn't.

* There is no need to say "(unofficial)". All (or nearly all?) of these are unofficial. My comment about it being unofficial wasn't a comment on the contents of the PR. It was a response to this comment: https://github.com/nodejs/nodejs.org/pull/1932#issuecomment-446316798

* It seems wrong to include Node.js Discord and not NodeSlackers.